AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -173,6 +173,7 @@ eatsmarter.de,inside-digital.de,hs.fi,vrt.be,manager-magazin.de,whats-on-netflix
 !
 !
 !
+jove.com##.css-ad08s1
```

Comment:
It's better to add the scriptlet for such sites, I did it directly.
Looks like the rule works, I approve this for now.